frappe/__init__.py

```diff
@@ -244,7 +244,7 @@ def init(site: str, sites_path: str = ".", new_site: bool = False) -> None:
 	local.session = _dict()
 	local.dev_server = _dev_server
 	local.qb = get_query_builder(local.conf.db_type or "mariadb")
-	local.qb.engine = get_qb_engine()
+	local.qb.get_query = get_query
```

Comment:
or is this not supposed to be "internal" anymore?

Reply:
@ankush 
I want people to use `frappe.qb.get_query` instead of `frappe.db.get_list`. I want it to be "public".
Is there any other way to make it public and not attach it to request locals?

Reply:
It's a replacement for `get_all` (sorta, get_list replacement needs perm checks) 

We can leave this here if it's gonna be public.